.github/workflows/pr-lint.yml

```diff
@@ -0,0 +1,17 @@
+name: "Lint PR"
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
+
+permissions:
+  pull-requests: read
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-22.04
```

Comment:
any reason not to use latest?

Reply:
We generally try to use exact dependencies for predictable runs. Happy to change if you feel strongly about this.